src/FileList/FileListHandler.h

```diff
@@ -37,11 +38,18 @@ class FileListHandler {
     void OnRegionFileInfoRequest(
         const CARTA::RegionFileInfoRequest& request, CARTA::RegionFileInfoResponse& response, ResultMsg& result_msg);
 
+    void StopGettingFileList() {
+        _stop_getting_file_list = true;
+    }
+    void SetProgressCallBack(const std::function<void(CARTA::Progress)>& progress_callback) {
```

Comment:
if we use `callback` as one word, it should be `SetProgressCallback`, shouldn't it?